README.md

```diff
@@ -1 +1,49 @@
-# To-do-list-App-R
\ No newline at end of file
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-do-list-App
+
+> This is my personnal App demo, feel free to check it out.
+
+
+## Built With
+
+- Html
+- Js
+
+
+
+ ## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link `https://github.com/camilevahviraki/To-do-list-app.git`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `https://github.com/camilevahviraki/To-do-list-app.git`.
+- go to the repository folder in your command prompt `To-do-list-app`
+
```

Comment:
### OPTIONAL 

- [ ] Kindly add commands to run this project locally, for example; npm install and npm start